plugins/com.contrastsecurity.ide.eclipse.core/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,3 @@
+#Properties file for com.contrastsecurity.ide.eclipse.core
```

Comment:
File corresponding to plugins/com.contrastsecurity.ide.eclipse.core/OSGI-INF/l10n/bundle_ja.properties